packages/commons/src/repositories/ReadModelRepository.ts

```diff
@@ -0,0 +1,32 @@
+import { EService, PersistentAgreement } from "pagopa-interop-models";
+import { Collection, MongoClient } from "mongodb";
+import { ReadModelDbConfig } from "../index.js";
+
+export type ReadModelRepository = {
+  eservices: Collection<{
+    data: EService | undefined;
+    metadata: { version: number };
+  }>;
+  agreements: Collection<{
+    data: PersistentAgreement;
```

Comment:
not related to this PR, but we should use the same naming conventions for types (`EService` vs `PersistentAgreement`).
I would ask to open a task for this change

Reply:
Do you mean that `PersistenAgreement` should be renamed to `Agreement`? I think that we used the old naming convention but I could open a task to change the name

Reply:
We can discuss the preferred naming convention, as long as all types follow the same one (all with `Persistent*` or all without)

Reply:
https://pagopa.atlassian.net/browse/IMN-79